gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskDtoBuilder.java

```diff
@@ -42,50 +41,70 @@ public TaskDto getTaskDto(OffsetDateTime timestamp) {
     }
 
     public List<TaskDto> getListTasksDto(LocalDate businessDate) {
```

Comment:
Generally speaking, I think you may add some blank lines in the code to air it and make its reading easier. At least that's my feeling.

For example, adding a blank line between current lines 49 and 50 to create a first block of code for initialization of datetime objects and a second block of code for tasks initialization.
Or adding a blank line between lines 90 and 91 to separate the `if` statement containing `return`s and the initialization of the variables.